lib/main.dart

```diff
@@ -51,6 +55,8 @@ class MyApp extends StatelessWidget {
   }
 }
 
+final credentialRepository = CredentialRepositoryImpl(getIt.get<ApiService>());
```

Comment:
```suggestion
final credentialRepository = getIt.get<CredentialRepository>();
```

Reply:
I refactored this in [here](https://github.com/nimblehq/ic-flutter-taher-toby/pull/50/files#:~:text=final%20_credentialRepository%20%3D%20getIt.get%3CCredentialRepository%3E()%3B) 🙏